internal/resources/credential/constants.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package credential
+
+const (
+	NameKey              = "name"
+	specKey              = "spec"
+	statusKey            = "status"
+	capabilityKey        = "capability"
+	providerKey          = "provider"
+	dataKey              = "data"
+	genericCredentialKey = "generic_credential"
+	// nolint:gosec
```

Comment:
Reason for this no lint?

Reply:
The use of credential makes the linter think it is hardcode credentials

Reply:
Why do we need these as hardcode credentials?

Reply:
I am not using hardcoded credentials, it just the schema key value